tracer/build/_build/Build.Steps.cs

```diff
@@ -298,7 +301,7 @@ partial class Build
         {
             if (IsWin)
             {
-                foreach (var architecture in new[] {"win-x86", "win-x64"})
+                foreach (var architecture in new[] { "win-x86", "win-x64" })
```

Comment:
Can we use the new field you added above to avoid repetition?
```suggestion
                foreach (var architecture in WafWindowsArchitectureFolders)
```